easybuild/easyconfigs/c/CD-HIT/CD-HIT-4.8.1-foss-2020a.eb

```
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'MakeCp'
+
+name = 'CD-HIT'
+version = '4.8.1'
+
+homepage = 'http://weizhong-lab.ucsd.edu/cd-hit/'
+description = """ CD-HIT is a very widely used program for clustering and 
+ comparing protein or nucleotide sequences."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
```

CD-HIT supports OpenMP and it seems that it could be moved down to the GCC toolchain, as it does not need either MPI or BLAS/LAPACK
```suggestion
toolchain = {'name': 'GCC', 'version': '9.3.0'}
toolchainopts = {'openmp': True}
```